build.gradle

```diff
@@ -85,8 +85,11 @@ sourceSets {
 }
 
 dependencies {
-    structuresCompile sourceSets.api.output
-    blockOutCompile sourceSets.api.output
+    // structuresCompile sourceSets.api.output
```

Comment:
This will remove the dependency of structures on the API, you will Need this later, as some structures classes check for hut blocks. Please Keep.